bin/docker-push

```diff
@@ -9,6 +9,9 @@ if [ -z "$DOCKERTAG" ];                     then echo DOCKERTAG must be set; exi
 if [ -z "$CI_SHA1" ];                       then echo CI_SHA1 must be set; exit 1; fi
 if [ -z "$CI_BUILD_NUM" ];                  then echo CI_BUILD_NUM must be set; exit 1; fi
 
+# support overriding "latest"
+LATEST=${LATEST:-latest}
```

Comment:
Quick note here - Can we consider a change from `LATEST` to `DOCKER_LATEST_TAG` as the env var? Since these are global I wanted to make sure the vars are clear where their used.

Reply:
sure that's easy